app/views/foreman_tasks/tasks/show.html.erb

```diff
@@ -15,9 +15,4 @@
   switcher_item_url: foreman_tasks_task_path(:id => ':id')
 ) %>
 
-
-<div class="task-details" id="foremanTaskDetails">
```

Comment:
We've got some css styles on `task-details` namely `.task-details pre { ... }`.
I'd choose an easy option and just leave the div, without the `id=""` part and put the mount inside.
But we can change css, or add the class into the component itself.

Reply:
I changed the css so it will be more specific, thanks for the suggestion!